Source/JavaScriptCore/inspector/remote/RemoteConnectionToTarget.h

```diff
@@ -50,7 +50,7 @@ typedef Vector<Function<void ()>> RemoteTargetQueue;
 class RemoteConnectionToTarget final : public ThreadSafeRefCounted<RemoteConnectionToTarget>, public FrontendChannel {
 public:
 #if PLATFORM(COCOA)
-    RemoteConnectionToTarget(RemoteControllableTarget*, NSString* connectionIdentifier, NSString* destination);
+    RemoteConnectionToTarget(Ref<RemoteControllableTarget>, NSString* connectionIdentifier, NSString* destination);
```

Comment:
`Ref<RemoteControllableTarget>&&`. We don't pass smart pointers by value to avoid ref-counting churn.